### PR TITLE
Moves RCDs and RPDs to Protolathes + Techwebs

### PR DESCRIPTION
**Pull request by BurgerLUA**
## About The Pull Request
Removes RCDs, RPDs, and RCD ammo from Hacked Autolathes and adds them to techwebs. They can be researched for 2000 points after industrial engineering is researched. RPDs can be accessed at science and engineering lathes, while RCDs and RCD ammo can be accessed at engineering lathes.

image

## Why It's Good For The Game
RCDs are exceptionally powerful for how easy it is to print one from a lathe. You don't even need insulated gloves to get one; you just need to print one from a hacked lathe, fill it with materials, and boom you can deconstruct any door or wall at will without having to use insulated gloves or a welding helmet. With the introduction of public auto lathes, it is even easier to get them.

## Changelog
:cl: BurgerBB
balance: RCDs, RPDs, and RCD Ammo are removed from hacked autolathes, and brought to protolathes.
/:cl:

### CODE REVIEW

**Comment by RogueSteampunker** [comment]
>don't need gloves

Considering that the autolathe is one of the only machines in the game that needs you to cut the wires instead to hack it, that seems rather false.

Second, RCDs naturally need materials to function, and the ones you can print don't start with any. You aren't getting far with what there is, and anything major you can do with it like breaking into the bridge can and will get you bwoinked.

And honestly, making the pr after you let the poll run for only an hour? Yikes

File: code/modules/research/techweb/all_nodes.dm [hunk 1]
```diff
@@ -167,6 +168,15 @@
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 6000)
 	export_price = 5000
 
+/datum/techweb_node/rapid_construction
+	id = "rapid_construction"
+	display_name = "Rapid Construction"
+	description = "Quick and almost miraculous tools that let you build things fast and easy."
+	prereq_ids = list("engineering")
+	design_ids = list("rcd","rpd","rcd_ammo")
```
Comment (Trilbyspaceclone) [inline code/modules/research/techweb/all_nodes.dm:176]: Were is rcd_ammo_large?

**Comment by BurgerLUA** [comment]
> > don't need gloves
> 
> Considering that the autolathe is one of the only machines in the game that needs you to cut the wires instead to hack it, that seems rather false.
> 
> Second, RCDs naturally need materials to function, and the ones you can print don't start with any. You aren't getting far with what there is, and anything major you can do with it like breaking into the bridge can and will get you bwoinked.
> 
> And honestly, making the pr after you let the poll run for only an hour? Yikes

A few things.

You can easily hack an autolathe without gloves using a multitool by pulsing the wires then cutting the one that makes the blue light glow when pulsing it. There are no electricity wires when you pulse something so it's completely safe without gloves if you know what you're doing.

The RCD is a powerful tool. Just because it's against the rules to be a cunt with the RCD does not mean that it should be easy to get. It's against the rules to randomly kill someone but that doesn't mean that everyone should be allowed to carry 9mm pistols. The rules should have very little impact on how hard it is to obtain something.

I made the PR because it was an easy thing to make. It's not like it will get merged in a short period, it is to show people EXACTLY what the balance would be like. It's a courtesy but somehow you're turning that courtesy into something malicious. I'd honestly appreciate if you didn't come here like an asshole not knowing what you're talking about and assuming that just because I made the extra effort to make a PR, it means that I don't give a shit what people think or whatever the fuck it is you're implying.

**Comment by ghost** [comment]
> You can easily hack an autolathe without gloves using a multitool by pulsing the wires then cutting the one that makes the blue light glow when pulsing it. There are no electricity wires when you pulse something so it's completely safe without gloves if you know what you're doing

When you pulse the wire which electrifies the machine, it will be electrified for 30 seconds. The person who pulsed the wire gets zapped. If you cut the wire, it stays electrified until the wire is mended. You are never truly 'safe' when you hack something unless you got insulated gloves.

edit: apparently autolathes don't have an electrify wire. Maybe you should fix that instead burger

**Comment by DeltaFire15** [comment]
dtfe..
Tve autolathe has NO electrify wire
The only wires that do anything are the 'hack' and the 'disable' wire
Its completely and utterly safe to do even without insuls.
(Apparently getting fixed soonTM by a different PR)

**Comment by ghost** [comment]
> dtfe..
> Tve autolathe has NO electrify wire
> The only wires that do anything are the 'hack' and the 'disable' wire
> Its completely and utterly safe to do even without insuls.
> (Apparently getting fixed soonTM by a different PR)

Oh really? I never realized. Anyhow, multitools don't save you from electrifying certain machines

My bad, sorry burger

**Comment by Poojawa** [comment]
Public Autolathes are a sin that shouldn't be a thing.

Remove their braindead easy material fill to only using RCD ammo and they're balanced again. Remove their efficiency discounts/up the cost for them if it's really a serious issue.

**Comment by silicons** [comment]
sorry not a fan for this, i'm open to other rcd nerfs/rpd nerfs (rpds are able to be abused pretty badly and the only thing stopping people is admins telling them to stop which is a bad thing).

**Pull request closed without merging**